File: java/src/main/java/com/iteso/nintendo/Thunderbolt.java
package com.iteso.nintendo;

/**
 * Created by rvillalobos on 2/24/18.
 */
public class Thunderbolt implements Attack {
    private int attackDamage;

    public String attack() {
        return "Thunderbolt";
    }

    public int getAttackDamage() {
        return attackDamage;
    }

    public void setAttackDamage(int attackDamage) {
        this.attackDamage = attackDamage;
    }
}

File: python/src/main/python/Thunderbolt.py
from Attack import *

class Thunderbolt(Attack):

    def __init__(self, damage = 90):
        self.damage = damage

    def performAttack(self):
        return "Thunderbolt"

    def getAttackDamage(self):
        return self.damage

    def setAttackDamage(self, damage):
        self.damage = damage